### PR TITLE
Add valid arguments to -sanitize-coverage: pc-table and inline-8bit-counter.

### PR DESCRIPTION
**Pull request by thetruestblue**
As is, you cannot set all default fuzzer options directly using -sanitize-coverage. Because of this you can use default fuzzer sanitize-coverage args, or a limited number coverage options.

This PR adds pc-table and inline-8bit-counter as valid args to be parsed for sanitize-coverage flag. These are default fuzzer options -- and will allow customizing sanitizer-coverage in variations of fuzzer defaults. (i.e. w/o pc-table enabled).

In upstream clang the option -fno-sanitize-coverage exists to disable various coverage options. Swift currently does not have a convention of using excluding args for sanitizer flags. This aims to limit the need for a new flag by inclusively setting desired coverage up to current fuzzer defaults.

Corresponding swift-driver change: https://github.com/apple/swift-driver/pull/1604

rdar://127881891

### CODE REVIEW

**Comment by thetruestblue** [comment]
@swift-ci test

**Comment by thetruestblue** [comment]
Using:
apple/swift-driver#1604
@swift-ci smoke test

**Comment by thetruestblue** [comment]
Using:
https://github.com/apple/swift-driver/pull/1604
@swift-ci smoke test

**Comment by thetruestblue** [comment]
Using:
https://github.com/apple/swift-driver/pull/1604
@swift-ci smoke test

File: test/Driver/sanitize_coverage.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // REQUIRES: asan_runtime
 
 // Try some options
-// RUN: %swiftc_driver -driver-print-jobs -sanitize-coverage=edge,indirect-calls,trace-bb,trace-cmp,8bit-counters  -sanitize=address %s | %FileCheck -check-prefix=SANCOV_EDGE_WITH_OPTIONS %s
+// RUN: %swiftc_driver -driver-print-jobs -sanitize-coverage=edge,indirect-calls,trace-bb,trace-cmp,8bit-counters,inlnie-8bit-counters,pc-table  -sanitize=address %s | %FileCheck -check-prefix=SANCOV_EDGE_WITH_OPTIONS %s
```
Comment (usama54321) [inline test/Driver/sanitize_coverage.swift]: inline*

Comment (thetruestblue) [reply test/Driver/sanitize_coverage.swift]: Already fixed and re-running test

**Comment by usama54321** [comment]
Reading from the docs, it seems like pc-table requires one other flag too. If clang throws an error/shows a warning if one of the other flags is missing, I would recommend replicating that here as well. Otherwise, LGTM
https://clang.llvm.org/docs/SanitizerCoverage.html

**Review by usama54321 — APPROVED** [review]

**Comment by thetruestblue** [comment]
I think you're referring to : `-fsanitize-coverage=inline-8bit-counters` or no? which I also added in this PR

**Review by artemcm — APPROVED** [review]
Looks good, thanks!

**Pull request merged**